src/zappy_server_src/core/cmake_error.c

```diff
@@ -0,0 +1,14 @@
+/*
+** EPITECH PROJECT, 2022
+** Project
+** File description:
+** cmake_error
+*/
+
+/// \file src/zappy_server_src/core/cmake_error.c
+
+/// \brief Do nothing
+void cmake_error(void)
+{
+    return;
+}
```

Comment:
Useless file ?

Reply:
Since our dear @LucasTesnier didn't want to upgrade his CMake version to 3.19 or upper we had to to make a hollow library in order to mimic an `INTERFACE` library.
In fact, the `zappy_entity_interface` library will be used by the `pack` library in order to simplify header dependecy between the server core and the pack